innocentEdosa/QUESTIONER
# #163073131 Setup Api dependencies

#### What does this PR do?
This PR install all Api dependencies

#### Description of Task to be completed?
npm install babel 
npm install express
npm install nodemon, jasmine, express-validator, nyc, eslint

#### How should this be manually tested?
clone the repo
checkout to ch-setup-api-dependencies-163073131 branch
check package.json file in editor

#### Any background context you want to provide?
These third party packages would help the api run better

#### What are the relevant pivotal tracker stories?
163073131

#### Screenshots
![h](https://user-images.githubusercontent.com/45862608/50899808-816f4d00-1413-11e9-9c57-700b7019b797.PNG)

#### Questions
can i go ahead with merging
